<launch>
    <arg name="sim" default="False" />
    <node name="bounding_box" pkg="object_detection" type="detect_peach.py" output="screen">
        <param name="~sim" value="$(arg sim)"/>
    </node>
</launch>